.buildkite/hooks/pre-command

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-eval "$(ejson2env .buildkite/env/secrets.ejson)"
+# eval "$(ejson2env .buildkite/env/secrets.ejson)"
```

Comment:
seems unrelated change? (also how about removing the dir as well?)

Reply:
ah, I'd prefer to do this as separate pr.

Reply:
I planned to open another PR for it. maybe I make it happen in this PR as well

Reply:
okay! I will open it when this one merge 👻

Reply:
the another PR is here https://github.com/solana-labs/solana/pull/30653 🪖